opendevin/controller/agent_controller.py

```diff
@@ -155,12 +155,10 @@ async def on_event(self, event: Event):
             await self.set_agent_state_to(event.agent_state)  # type: ignore
         elif isinstance(event, MessageAction):
             if event.source == EventSource.USER:
-                logger.info(event, extra={'msg_type': 'OBSERVATION'})
```

Comment:
Action is logged as OBSERVATION. Intended?

Reply:
This is a message from the USER, so, to AGENT, it is OBSERVATION. So yes, this is intended.

Reply:
```
OBSERVATION
**MessageAction** (source=EventSource.USER)
```
isn't the source enough?

Reply:
I think it doesn't say "Observation" in agent's history. It's just the formatting in the log, for us humans.

Reply:
Yep - @enyst is right. The OBSERVATION here is just here for colored logging for human.

Reply:
`extra={'msg_type': 'ACTION'})` also colors.

Reply:
They are different colors for visual distinction

Reply:
Should add USER_ACTION type.